src/nri/sampler.go

```diff
@@ -269,12 +269,25 @@ func pids(pids *biz.Pids) []entry {
 }
 
 func blkio(bio *biz.BlkIO) []entry {
+	var totalBytes *float64
+
+	if bio.TotalReadBytes != nil || bio.TotalWriteBytes != nil {
+		sum := 0.0
+		if bio.TotalReadBytes != nil {
+			sum += *bio.TotalReadBytes
+		}
+		if bio.TotalWriteBytes != nil {
+			sum += *bio.TotalWriteBytes
+		}
+		totalBytes = &sum
+	}
+
 	return []entry{
 		metricIOTotalReadCount(bio.TotalReadCount),
 		metricIOTotalWriteCount(bio.TotalWriteCount),
 		metricIOTotalReadBytes(bio.TotalReadBytes),
```

Comment:
shuoln't we check if bio.TotalReadBytes is nil?

Reply:
(same comment for all the functions)

Reply:
I fear we might introduce a nil pointer exception

Reply:
Updated.